volmdlr/step.py

```diff
@@ -227,35 +229,34 @@ def create_graph(self, draw=False, html=False):
         #     nx.draw_networkx_labels(F, pos, labels)
         #     # ------------------------------------
         #
-        # if html:
-        #
-        #     env = Environment(
-        #         loader=PackageLoader('powertransmission', 'templates'),
-        #         autoescape=select_autoescape(['html', 'xml']))
-        #     template = env.get_template('graph_visJS.html')
-        #
-        #     nodes = []
-        #     edges = []
-        #     for label in list(labels.values()):
-        #         nodes.append({'name': label, 'shape': 'circular'})
-        #
-        #     for edge in G.edges:
-        #         edge_dict = {}
-        #         edge_dict['inode1'] = int(edge[0]) - 1
-        #         edge_dict['inode2'] = int(edge[1]) - 1
-        #         edges.append(edge_dict)
-        #
-        #     options = {}
-        #     s = template.render(
-        #         name=self.stepfile,
-        #         nodes=nodes,
-        #         edges=edges,
-        #         options=options)
-        #
-        #     with open('graph_visJS.html', 'wb') as file:
-        #         file.write(s.encode('utf-8'))
-        #
-        #     webbrowser.open('file://' + os.path.realpath('graph_visJS.html'))
+        if html:
+
+            env = Environment(
```

Comment:
Better use template substitution as https://github.com/Dessia-tech/volmdlr/blob/1304748eb8d41dd3c1b6220a0fb9d5a7659ef27d/volmdlr/core.py#L1486

Reply:
Fixed in https://github.com/Dessia-tech/volmdlr/pull/190/commits/3c49b6ae958e2591dd326b81e3fd694d4084c141